### PR TITLE
Added suse-vale-styleguide VSCode extension

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: extensions/vale-styleguide/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Installation
+
+For this extension to work, meet the following requirements:
+
+1. Download the `vale` binary from
+   [github.com/errata-ai/vale/releases](https://github.com/errata-ai/vale/releases),
+   unpack it and copy to a directory in your `$PATH`.
+2. Install the `vale-vscode` VSCode extension from the VSCode market place
+    ```
+    ext install ChrisChinchilla.vale-vscode
```
Comment (tomschr) [inline extensions/vale-styleguide/README.md:17]: I don't have the `ext` executable nor do I know where to get it. Perhaps for others is also a big problem. However, most have installed `code`.

Would it be an option to install the extension like this:

```
code --install-extension ChrisChinchilla.vale-vscode
```

I've tried it and it gives me this output:

```
$ code --install-extension ChrisChinchilla.vale-vscode
Installing extensions...
Installing extension 'chrischinchilla.vale-vscode'...
(node:31365) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
(Use `code --trace-deprecation ...` to show where the warning was created)
Extension 'chrischinchilla.vale-vscode' v0.17.3 was successfully installed.
```

```suggestion
    code --install-extension ChrisChinchilla.vale-vscode
```

File: extensions/vale-styleguide/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ```
+3. Install this openSUSE.suse-vale-styleguide extension from the VSCode market
+   place
+    ```
+    ext install openSUSE.suse-vale-styleguide
```
Comment (tomschr) [inline extensions/vale-styleguide/README.md:22]: Same here:

```suggestion
    code --install-extension openSUSE.suse-vale-styleguide
```

File: extensions/vale-styleguide/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "suse-vale-styleguide",
+  "displayName": "SUSE Vale rules",
+  "description": "Vale rules for SUSE styleguide",
+  "version": "0.1.0",
```
Comment (tomschr) [inline extensions/vale-styleguide/package.json:5]: Do you raise the version afterwards? I think this change would qualify for a higher version (maybe 0.2.0?)

**Review by tomschr — CHANGES_REQUESTED** [review]
@tbazant That's very cool! Thanks Tomas! :+1: 
I have some small suggestions, hope you find them useful.

File: extensions/vale-styleguide/.vscode/extensions.json
Comment (tomschr) [inline extensions/vale-styleguide/.vscode/extensions.json:1]: Would it make sense to add further extensions that you need to this file?

See an example from https://stackoverflow.com/a/46656717

File: extensions/vale-styleguide/package.json (hunk 3 above)
Comment (tbazant) [reply extensions/vale-styleguide/package.json:5]: it's the initial version of the extension, why not 0.1.0?

File: extensions/vale-styleguide/.vscode/extensions.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
+	"recommendations": [
+		"dbaeumer.vscode-eslint"
```
Comment (tomschr) [inline extensions/vale-styleguide/.vscode/extensions.json:5]:
```suggestion
		"dbaeumer.vscode-eslint",
		"ChrisChinchilla.vale-vscode"
```

File: extensions/vale-styleguide/package.json (hunk 3 above)
Comment (tomschr) [reply extensions/vale-styleguide/package.json:5]: Oh, I see! In that case, 0.1.0 is perfectly fine.

**Pull request merged**